selfdrive/controls/lib/drive_helpers.py

```diff
@@ -13,7 +13,7 @@
 V_CRUISE_UNSET = 255
 V_CRUISE_INITIAL = 40
 V_CRUISE_INITIAL_EXPERIMENTAL_MODE = 105
-IMPERIAL_INCREMENT = 1.6  # should be CV.MPH_TO_KPH, but this causes rounding errors
+IMPERIAL_INCREMENT = round(CV.MPH_TO_KPH, 1)  # round here to avoid rounding errors incrementing set speed
```

Comment:
Don't we still want to switch to the unrounded value? It's debatable if this is still to get around a bug or intentional

Reply:
If we don't ever switch to the unrounded value what should we do instead of this change? If the reason we need this specific value is because ecus are using a rounded conversion constant, this seems like a cleaner way to get that value.